eo-maven-plugin/src/main/java/org/eolang/maven/tojos/ForeignTojo.java

```diff
@@ -404,4 +404,21 @@ public boolean equals(final Object other) {
     public int hashCode() {
         return Objects.hash(this.delegate);
     }
+
+    /**
+     * Return the attribute from the tojo.
+     * @param attribute The attribute from ForeignTojos.Attribute.
+     * @return The attribute.
+     */
+    private String attribute(final ForeignTojos.Attribute attribute) {
+        final String attr = this.delegate.get(attribute.key());
+        if (attr == null) {
+            throw new AttributeNotFoundException(
+                String.format(
+                    "There is no '%s' attribute in the tojo", attribute
```

Comment:
@Yanich96 maybe let's move the message itself to the `AttributeNotFoundException` and pass only attribute here?